cdq-analysis/pkg/util.go

```diff
@@ -170,8 +170,11 @@ func ConvertGitHubURL(URL string, revision string, context string) (string, erro
 	if err != nil {
 		return "", &InvalidURL{URL: URL, Err: err}
 	}
+	if !strings.Contains(url.Host, "github") {
```

Comment:
QQ: how did you test this change? 

This is fine with CLI/test suite import cases. Does it also work with Konflux UI? Because that uses CDQ first before going to Component controller... 🤔

Reply:
Oh ok. TBH I haven't changed it with the Konflux UI. Having in mind what you mention, I think is better to include this change only on the component controller side right?

Reply:
@maysunfaisal following your suggestion I've moved the functionality to the `component_controller` side.